starknet/src/utils/into.cairo

```diff
@@ -0,0 +1,35 @@
+use array::{ArrayTrait, SpanTrait};
+use option::OptionTrait;
+use traits::Into;
+use integer::u256_from_felt252;
+use starknet::{ContractAddress, EthAddress};
+
+impl Felt252SpanIntoU256Array of Into<Span<felt252>, Array<u256>> {
+    fn into(self: Span<felt252>) -> Array<u256> {
+        let mut self = self;
+        let mut arr = ArrayTrait::<u256>::new();
+        loop {
+            match self.pop_front() {
+                Option::Some(num) => {
+                    arr.append((*num).into());
+                },
+                Option::None(_) => {
+                    break ();
+                },
+            };
+        };
+        arr
+    }
+}
+
+impl ContractAddressIntoU256 of Into<ContractAddress, u256> {
```

Comment:
Same, basically here you are using the fact that self has an `IntoFelt252` impl, so you can make it generic over anything that has a `IntoFelt252` impl I believe :)

```rust
impl<T, impl TIntoFelt252: Into<T, felt252>> TIntoU256 of Into<T, u256>
```

Reply:
yep agree